src/components/Map/Map/Map.tsx

```diff
@@ -12,17 +11,57 @@ const circleOptions = (radius) => ({
     draggable: false,
     editable: false,
     visible: true,
-    radius,
     zIndex: 1,
 });
 
-export const Map = React.forwardRef((props, ref) => {
-    const { defaultArea, markers, mapContainerStyle, defaultHighlight, ...rest } = props;
-    const mapRef = ref || React.createRef();
-    const [highlightFeatures, setHighlightFeatures] = React.useState(null);
+type CircularMarker = {
```

Comment:
Should this rather go inside `src/types/`?

Reply:
ok, I can move it.

Reply:
looking at it closer... we don't have many global type declarations in OneUI. They more local and exported only when needed. I would keep it as it is for now.
If we move it to a separate file, I would argue to move all types from LocationSelector/utils.ts as well. And then look around if there are some others. Then the question is how do we expose these outside of OneUI, which we already have a ticket for. So I would wait for that, and do it all at once.